### PR TITLE
Add option to Ignore Max Height on AutoExpand

### PR DESCRIPTION
**Pull request by bigjonroberts**
This was exposed via the API, but not with the options.  I'm submitting a pull request for the docs as well to document the new option.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
-  - 0.10
+- 0.1
 before_script:
-  - npm install -g grunt-cli
-  - "phantomjs --version"
+- npm install -g grunt-cli
+- phantomjs --version
 notifications:
   email:
-    - sam@sceditor.com
```
Comment (brunoais) [inline .travis.yml]: Why are you removing this?

File: src/lib/SCEditor.js [hunk 2]
```diff
@@ -355,7 +355,7 @@ define(function (require) {
 				}
 
 				if (options.autoExpand) {
-					base.expandToContent();
+					base.expandToContent(options.ignoreMaxHeight);
```
Comment (brunoais) [inline src/lib/SCEditor.js]: Why didn't you change
https://github.com/samclarke/SCEditor/blob/master/src/lib/SCEditor.js#L491
too?

File: .travis.yml (hunk 1 above)
Comment (bigjonroberts) [inline .travis.yml]: I did that for my personal builds off of my fork - should not have been included in pull request.  I'll revert that change.

File: src/lib/SCEditor.js (hunk 2 above)
Comment (bigjonroberts) [inline src/lib/SCEditor.js]: Thanks, I missed that one.  I'll add it and test it. Then add a comment for review after I push again.

**Comment by brunoais** [comment]
I'm still waiting for the commit fixes

**Comment by samclarke** [comment]
Closing as needs fixes.

**Pull request closed without merging**